harness/determined/deploy/aws/templates/secure.yaml

```diff
@@ -3,46 +3,46 @@ Description:  This template deploys a VPC, with a public and private subnet. It
 Mappings:
   RegionMap:
     ap-northeast-1:
-      Master: ami-005021e678ab162ea
-      Agent: ami-039e09c6582ec61c5
-      Bastion: ami-04fe60822d08387bb
+      Master: ami-09ff2b6ef00accc2e
+      Agent: ami-0aa0009aef18b0bf6
+      Bastion: ami-09ff2b6ef00accc2e
```

Comment:
when i ran `./refresh-ubuntu-amis.py bumpenvs.yaml` the bastion ami's didn't refresh. not sure what your run did differently.

Reply:
> I updated the release line of Ubuntu

Where is this (since I did run bump envs with your changes)?

Reply:
oh i see in tools/scripts